src/Argu.Dotnetcli.Tests/project.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "1.0.0-*",
+    "buildOptions": {
+        "debugType": "portable",
+        "emitEntryPoint": true,
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "Program.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+            "version": "1.0.0-preview2-*",
+            "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6" : { 
```

Comment:
it's a console app, you need to use `netcoreapp1.0` instead of `netstandard1.6`.
see [example console app with preview2](https://github.com/dotnet/netcorecli-fsc/tree/master/examples/preview2/console), add `.NETCore.App` too inside `frameworks.netcoreapp1.0` dependencies